roles/confluent.test/upgrade_testing/upgrade.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -e
+
+## Variables
+
+export END_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+export KSQL_INVALID_VERSION=5.4
+
+## Change directory 
+
+cd ..
+
+## Checkout starting branch
+
+echo "Checking out $START_BRANCH branch"
+git checkout $START_BRANCH
+
+## Run Molecule Converge on scenario
+
+echo "Running molecule converge on $SCENARIO_NAME"
+molecule converge -s $SCENARIO_NAME
+
```

Comment:
why not just do a 
```
cd ../..
```
here and not have to do it later

Reply:
To me this is a stylistic difference and really doesn't impact the code or it's readability.  I prefer the current format.